src/tests/common/control_topic/xi_cbor_codec_ct_server.c

```diff
@@ -112,6 +112,40 @@ void xi_cbor_codec_ct_server_encode( const xi_control_message_t* control_message
                             &err ),
                         &err );
 
+                    if ( NULL !=
+                         control_message->file_update_available.list[id_file]
+                             .download_link )
+                    {
+                        cn_cbor_map_put(
+                            file, cn_cbor_string_create(
+                                      XI_CBOR_CODEC_CT_STRING_FILE_DOWNLOADLINK
+                                          CBOR_CONTEXT_PARAM,
+                                      &err ),
+                            cn_cbor_string_create(
+                                control_message->file_update_available.list[id_file]
+                                    .download_link CBOR_CONTEXT_PARAM,
+                                &err ),
+                            &err );
+
+                        cn_cbor* cn_cbor_bool = CN_CALLOC_CONTEXT();
```

Comment:
I can see 2 new CN_CALLOC calls, but no new CN_FREE in those files. Where will these pointers be deallocated?

(I'm not looking for an exact answer, just making sure it's not leaky)